src/identityApiClient.ts

```diff
@@ -216,35 +234,42 @@ export default function IdentityAPIClient(this: IIdentityApiClient, mpInstance)
                 false
             );
         } catch (err) {
-            console.log('did i error?');
-            console.log('callback', callback);
             mpInstance._Store.identityCallInFlight = false;
-            invokeCallback(callback, HTTPCodes.noHttpCoverage, err);
+            invokeCallback(
+                callback as Callback,
+                HTTPCodes.noHttpCoverage,
+                err.toString()
```

Comment:
I think you can mimic [line 144](https://github.com/mParticle/mparticle-web-sdk/pull/948/files/8a2e225a3e69d97b437f21d03c9a156c8561fb32..e8be568fde477988b36ddc4ceb8ddd2a67132d23#diff-3e282016703704717fe073d29ecd32c1bd2505325205cfca20d18fd4c8003030R144-R151) and cast the error as type Error in order to set `err.message` here instead of `err.toString()`